apps/cowswap-frontend/src/legacy/hooks/useWrapCallback.ts

```diff
@@ -85,9 +71,6 @@ export async function wrapUnwrapCallback(
     })
     useModals && closeModals()
 
-    // Clean up form fields after successful wrap/unwrap
-    updateTradeState({ ...tradeState, inputCurrencyAmount: '', outputCurrencyAmount: '' })
-
```

Comment:
This is the fix.

After the wrap, the input amounts were reset, clearing the trade state, short-circuiting the flow from then on.